app/controllers/discover.js

```diff
@@ -121,6 +127,39 @@ export default ApplicationController.extend({
         });
     },
 
+    transformTypes(obj) {
```

Comment:
I'm not sure I see the point of this function, why wouldn't you just lowercase the keys whenever you use them?